ccdproc/image_collection.py

```diff
@@ -58,14 +58,19 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     hdu: str or int, optional
+         Name/Index of the extension, which specifies the extension to extract data
+         from all files in a given directory.
+         Default is 0.
+
     Raises
     ------
     ValueError
         Raised if keywords are set to a combination of '*' and any other
         value.
     """
     def __init__(self, location=None, keywords=None, info_file=None,
-                 filenames=None):
+                 filenames=None, hdu=0):
```

Comment:
😳 Guess who chose to call it `hdu` in `CCDData.read`...that would be me 😱 

I guess in the context of `ImageFileCollection` I was thinking that `hdu` was unclear because there are multiple files, there is an `hdus` iterator, and so I immediately wondered which `hdu` of the `hdus` in the collection `ic.hdu` would refer to.

Using `ext` would avoid that potential confusion...though it is entirely possible I'm the source of confusion.